wallet/db.h

```diff
@@ -40,7 +40,7 @@ struct db;
  * devtools/sql-rewrite.py needs to change as well, since they need to
  * generate identical names to work correctly.
  */
-#define SQL(x) NAMED_SQL( __FILE__ ":" stringify(__LINE__) ":" stringify(__COUNTER__), x)
+#define SQL(x) NAMED_SQL( __FILE__ ":" stringify(__COUNTER__), x)
```

Comment:
Hmm, I guess we can assume all compilers support __COUNTER__.  Should be fine.

Reply:
Yeah, the `__LINE__` macro didn't work across multiple compilers. I'm now simply using the raw string of the original query, so we could drop this again (or reinstate the `__LINE__` as a debugging hint when working on the rewriter).

I was hoping to shortcut the string comparison a bit by having shorter names, rather than comparing the full statements which are longer, but generating identifiers proved to be more difficult.